packages/pong-engine/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
maybe we would like to add another `.prettierrc` in this folder with this content
```
{
  "singleQuote": true,
  "trailingComma": "all"
}
```

So our autoformatters don't mess everything up?
Maybe there's a way to place it in the monorepo root?

Reply:
Also, maybe it'd be convenient to have a GitHub job that checks if the packages folder is properly styled/linted/builds as well?

Reply:
Yes good idea 👍 a benefit of monorepos is to share configuration also, I'll add a common prettier and eslint configuration.

Also I'll add an additional GI job for the packages, I also think is a good idea to test them in PRs

Reply:
Finally added:

- a common config for prettier. Not for eslint since webapp and transcendence-app have different configurations
- extract style checking to a independent GH action. Done like this because, apparently, there is no way to only install `devDependencies` at root level. This would be desirable so we wouldn't have to do the full `npm i` globally in each service action just to install prettier and run the style check. Maybe there is a better way to do it but don't want to waste more time on this :(
- create a new GH action for the packages